src/js/components/KeptEntry.jsx

```diff
@@ -61,34 +60,34 @@ var KeptEntry = React.createClass({
     event.preventDefault();
     this.unhighlight();
     var newIndex = parseInt(event.dataTransfer.getData("text/plain"), 10);
-    this.props.move(newIndex, this.props.key);
+    this.props.move(newIndex, this.props.itemIndex);
   },
 
   highlight: function() {
-    this.getDOMNode().querySelector(".panel").classList.add("targetted");
+    reactDom.findDOMNode(this.refs.panel).classList.add("targetted");
   },
 
   unhighlight: function() {
-    this.getDOMNode().querySelector(".panel").classList.remove("targetted");
+    reactDom.findDOMNode(this.refs.panel).classList.remove("targetted");
```

Comment:
Yup that true, 
Not sure if it's a bug or not, but it work as expected when applied on a DOM node (`<div ref="div" ` but not on React Component